api/envoy/config/trace/v3/zipkin.proto

```diff
@@ -68,4 +68,11 @@ message ZipkinConfig {
   // Optional hostname to use when sending spans to the collector_cluster. Useful for collectors
   // that require a specific hostname. Defaults to :ref:`collector_cluster <envoy_v3_api_field_config.trace.v3.ZipkinConfig.collector_cluster>` above.
   string collector_hostname = 6;
+
+  // If this is set to true, then Envoy will be treated as independent service. Server span will be created
+  // for downstream request and client span will be created for upstream request.
+  // If this is set to true, then the
+  // :ref:`start_child_span of router <envoy_v3_api_field_extensions.filters.http.router.v3.Router.start_child_span>`
+  // SHOULD be set to true also.
```

Comment:
Maybe add what happens if `start_child_span` is set to false?

Reply:
> Maybe add what happens if `start_child_span` is set to false?

If the `independent_proxy` is set to true and  `start_child_span` is set to false, then error trace chain will be generated in the sidecar mode.

```
server span (inbound sidecar) -> APP -> server span (outbound sidecar).
```